onnxruntime/core/providers/dml/DmlExecutionProvider/src/ExecutionProvider.h

```diff
@@ -5,6 +5,7 @@
 
 #include "GraphTransformer.h"
 #include "core/providers/dml/DmlExecutionProvider/inc/IWinmlExecutionProvider.h"
+#include "core/providers/dml/DmlExecutionProvider/src/IExecutionProvider.h"
 
```

Comment:
Do we really need this header? I think none of the change in this file use any of the struct from this header file.

Reply:
It is needed because `ExecutionProvider.h` is now included by files that don't themselves include `IExecutionProvider.h`. In general, headers should include everything they used in order to make them easily includable everywhere, but we don't do a very good job of that :D